### PR TITLE
Add CVO manifests 

### PR DESCRIPTION
**Pull request by zvonkok**
This PR labels the Dockefile for CVO inclusion.
Adds auto-generation of CVO manifests and adds image-references and ClusterOperator CR.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.15 as builder
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
```
Comment (ArangoGutierrez) [inline Dockerfile:2]: we can leave golang here, what really matters is to make this change in the downstream version

File: Dockerfile [hunk 2]
```diff
@@ -23,11 +23,16 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod=vendor -a
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM registry.svc.ci.openshift.org/ocp/4.7:base
```
Comment (ArangoGutierrez) [inline Dockerfile:26]: same

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]
just a nit, could you rename the files to a more descriptive name , all files are  
 cvo-manifests/0001_specialresource_cvo_manifest.yaml  0001 to 0011....
better to have 00XX_specialresource_KUBERNETESOBJECT.yaml
example
0001_specialresource_CustomResourceDefinition.yaml

**Comment by dagrayvid** [comment]
> just a nit, could you rename the files to a more descriptive name , all files are
> cvo-manifests/0001_specialresource_cvo_manifest.yaml 0001 to 0011....

Can we rename `cvo-manifests` to just `manifests`, to match other CVO managed operators?

File: Makefile [hunk 3]
```diff
@@ -138,6 +141,9 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
+csplit:
```
Comment (dagrayvid) [inline Makefile]: Is this empty make target intentional?

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Comment by dagrayvid** [comment]
Thanks for the fixes.
/lgtm

**Review by dagrayvid — APPROVED** [review]

**Pull request closed without merging**